test-sbt/jvm/src/test/scala/zio/test/runner/ZTestFrameworkSpec.scala

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2019 John A. De Goes and the ZIO Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package zio.test.runner
+
+import java.util.concurrent.atomic.AtomicReference
+
+import sbt.testing._
+import zio.BaseCrossPlatformSpec
+import zio.test.{ assert, suite, test, DefaultRunnableSpec, Predicate, TestAspect }
+import ZTestFrameworkSpec._
+
+import scala.collection.mutable.ArrayBuffer
+
+class ZTestFrameworkSpec extends BaseCrossPlatformSpec {
```

Comment:
Seeing as how this is so foundational, I'd rather not rely on any testing framework, but rather do what ZIO test does (main function with asserts). Removes another dependency.